### DISCUSSION TITLE
Bem-vindos ao fórum pix-api!

### DISCUSSION THREAD

**Discussion by ninrod** [bacen/pix-api#242] [General]
# 👋 Bem-vindos!
  Estamos testando as "discussões do github" como um lugar para interação com membros da comunidade. Esperamos que você:
  * Faça perguntas.
  * Compartilhe ideias e projetos relacionados ao Pix e à API.
  * Interaja com outros membros da comunidade.
  * Mantenha a mente aberta para outras opiniões. Lembre-se de que o Pix é sistema criado pelo Banco Central para benefício da sociedade brasileira. O Pix pertence a todos os brasileiros.
  * providencie comentários sobre funcionalidades, melhorias futuras, elogios, críticas e caminhos alternativos.

**Comment by ggarlera** [comment]
Olá...  gostaria de saber se alguém tem uma tabela de retorno de erros do PIX ? Se possui algum padrão já definido do BC ou se no arranjo já possuem algo?

**Comment by GuerreiroPUSHINPAY** [comment]
Para todos ! Segue opção de PIX via API - pushinpay.com.br 
Melhores taxas do mercado
